lib/order.rb

```diff
@@ -1,18 +1,61 @@
+require 'csv'
+
+FILE_NAME = "../support/orders.csv"
+
```

Comment:
You should omit the `../` from this path so that `rake` can run. If you need to files individually, you should do so from the project root. For example:

```
$ ls
README.md  Rakefile  feedback.md  lib  seeds.rb  specs  support
$ ruby lib/order.rb
[ ... program output ... ]
```